src/components/venue-directory/VenueDirectory.jsx

```diff
@@ -127,53 +134,65 @@ export function VenueDirectory(props) {
           We're sowwy. 😞 <strong>We didn't find any venues for that search or combination of tags.</strong> <br/>We're indexing hundreds of venues per month! So, check back later or ask the community in <a href="https://discord.gg/gTP65VYcMj">our discord</a>!
         </div>
     }
-    
+
     { /* Favorites */ }
-    { isLoadingOrLoadedWithResults(filteredFavorites) && 
+    { isLoadingOrLoadedWithResults(filteredFavorites) &&
     <div className="aether-venues__venues aether-venues__favourite-venues">
       <details open>
         <summary><h2>Favorites</h2></summary>
-        { listView 
+        { listView
           ? <VenueList venues={filteredFavorites} />
           : <VenueStrip venues={filteredFavorites} /> }
       </details>
     </div> }
 
     { /* Open */ }
-    { isLoadingOrLoadedWithResults(filteredOpen) && 
+    { isLoadingOrLoadedWithResults(filteredOpen) &&
     <div className="aether-venues__venues aether-venues__opennow">
       <details open>
         <summary><h2>Open now</h2></summary>
-        { listView 
+        { listView
           ? <VenueList venues={filteredOpen} />
           : <VenueStrip venues={filteredOpen} />
         }
       </details>
     </div> }
 
     { /* Newest */ }
-    { isLoadingOrLoadedWithResults(filteredNewest) && 
+    { isLoadingOrLoadedWithResults(filteredNewest) &&
     <div className="aether-venues__venues aether-venues__new-venues">
       <details open={!listView}>
         <summary><h2>Newest</h2></summary>
-        { listView 
+        { listView
           ? <VenueList venues={filteredNewest} />
           : <VenueStrip venues={filteredNewest} />
         }
       </details>
     </div> }
 
+    { /* Hiring */ }
+    { isLoadingOrLoadedWithResults(filteredHiring) &&
+    <div className="aether-venues__venues aether-venues__now-hiring">
+      <details open>
+        <summary><h2>Hiring</h2></summary>
+        { listView
+          ? <VenueList venues={filteredHiring} />
+          : <VenueStrip venues={filteredHiring} />
+        }
+      </details>
+    </div> }
```

Comment:
Might be best as a filter than it's own strip? Might simplify your changes a little too.